<launch>
    <arg name="directory" default="/home/junhakim/camlidar_data/"/>

    <arg name="n_cameras" default="1"/>
    <arg name="n_lidars" default="2"/>

    <arg name="image_name0" default="/0/image_raw"/>    
    
    <!--remap from="/os1_cloud_node/points" to="lidar0/velodyne_points"/-->

    <node pkg="camlidar_logger" name="camlidar_logger_node" type="camlidar_logger_node" output="screen">
        <param name="n_cameras" type="int" value="$(arg n_cameras)"/>
        <param name="n_lidars" type="int" value="$(arg n_lidars)"/>
        <param name="image_name0" type="string" value="$(arg image_name0)"/>
        <param name="directory" type="string" value="$(arg directory)"/>
    </node>

</launch>
